lib/motsu/Cargo.toml

```diff
@@ -2,7 +2,8 @@
 name = "motsu"
 description = "Unit Testing for Stylus"
 edition.workspace = true
-keywords.workspace = true
+categories = ["development-tools::testing", "cryptography::cryptocurrencies"]
+keywords = ["arbitrum", "ethereum", "stylus", "tests", "openzeppelin"]
```

Comment:
We're not including OZ as a keyword in the other packages, and I think is not the best practice to include company name(and it is already part of the package name and owners)

Reply:
It is not the part of the name "motsu", that is why I included it. But yeah, owner is already "openzeppelin". Will remove it

Reply:
We are allowed to have up to 5 keywords per crate. So added "unit-tests" for `motsu` and "integration-testing" for `e2e` instead of "openzeppelin". Those exact tags are frequently used at crates.io to describe corresponding crates.